### DISCUSSION TITLE
What are Send & Recv Speed?

### DISCUSSION THREAD

**Discussion by houmie** [masonr/yet-another-bench-script#66] [Q&A] [answered]
Although this might seem obvious, it is not clear to me.

When I run this script on a VPS, what does Send & Receive mean?

```
Provider        | Location (Link)           | Send Speed      | Recv Speed      | Ping
-----           | -----                     | ----            | ----            | ----
Clouvider       | London, UK (10G)          | 565 Mbits/sec   | 816 Mbits/sec   | 55.7 ms
```

**Is Send speed the speed of packets that are sent from Clouvider to my VPS?
Or is it from my VPS to Clouvider?**

Many Thanks,

**Comment by Debcharon** [comment, marked as answer]
In this case, "Send Speed" represents the upload speed from your VPS to the Clouvider server, and "Recv Speed" represents the download speed from the Clouvider server to your VPS.

Send Speed: VPS --> Clouvider
Recv Speed: Clouvider --> VPS

**Reply by houmie** [reply]
Thank you
